modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java

```diff
@@ -81,29 +81,6 @@ protected OpenServiceBrokerExamples() { }
       System.setProperty("IBM_CREDENTIALS_FILE", "../../open_service_broker.env");
   }
 
-//
-// This class provides an example of how to use the Open Service broker service.
-//
-// The following configuration properties are assumed to be defined:
-//
-// OPEN_SERVICE_BROKER_URL=<Service broker's URL>
-// OPEN_SERVICE_BROKER_AUTH_TYPE=basic
-// OPEN_SERVICE_BROKER_USERNAME=<username>
-// OPEN_SERVICE_BROKER_PASSWORD=<password>
-// OPEN_SERVICE_BROKER_PLAN_ID=<The ID of the plan associated with the service offering>
-// OPEN_SERVICE_BROKER_RESOURCE_INSTANCE_ID=<The ID of a previously provisioned service instance>
-// OPEN_SERVICE_BROKER_SERVICE_ID=<The ID of the service being offered>
-// OPEN_SERVICE_BROKER_ACCOUNT_ID=<User's account ID>
-// OPEN_SERVICE_BROKER_BINDING_ID=<The ID of a previously provisioned binding for that service instance>
-// OPEN_SERVICE_BROKER_SPACE_GUID=<The identifier for the project space within the IBM Cloud platform organization>
-// OPEN_SERVICE_BROKER_APPLICATION_GUID=<GUID of an application associated with the binding>
-// OPEN_SERVICE_BROKER_ORGANIZATION_GUID=<The IBM Cloud platform GUID for the organization under which the service instance is to be provisioned>
-//
-// These configuration properties can be exported as environment variables, or stored
-// in a "credentials" file and then:
-// export IBM_CREDENTIALS_FILE=<name of credentials file>
-//
-
```

Comment:
Why this has been removed?

Reply:
It was a duplicated section within the file.
The information still available in the file, you can check it here:
https://github.com/Andras-Csanyi/platform-services-java-sdk/blob/f07da2eca6c0622ef07e623fdbf331a36377e6f0/modules/examples/src/main/java/com/ibm/cloud/platform_services/open_service_broker/v1/OpenServiceBrokerExamples.java#L42

Reply:
I see, thanks!